commcare_connect/templates/prelogin/home.html

```
@@ -99,8 +99,8 @@
       <a href="/portfolio" data-route="/portfolio">Portfolio</a>
       <a href="/frontline-network" data-route="/frontline-network">Frontline Network</a>
       <div class="nav-cta">
-        <a href="{{ app_login_url }}" class="btn btn-ghost">Sign In</a>
-        <a href="https://connect.dimagi.com/accounts/signup/" class="btn btn-primary" target="_blank" rel="noopener">Sign Up</a>
+        <a href="{% url "account_login" %}" class="btn btn-ghost">Sign In</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)